src/App.css

```diff
@@ -19,6 +19,35 @@
   color: white;
 }
 
+.user-panel {
```

Comment:
Maybe this could be .UserPanel to keep consistency.

Also consider moving these styles to a new UserPanel component